### PR TITLE
Update hkex.com.hk

### PR DESCRIPTION
**Pull request by Marco129**

### CODE REVIEW

File: src/chrome/content/rules/hkex.com.hk.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--
+	Invalid certificate:
+		en-rules.hkex.com.hk
+		www.iporesults.hkex.com.hk
+
+	No working URL known:
+		sc.hkex.com.hk
```
Comment (J0WI) [inline src/chrome/content/rules/hkex.com.hk.xml]: `http://sc.hkex.com.hk/TuniS/www.hkex.com.hk/chi/index_c.htm`

File: src/chrome/content/rules/hkex.com.hk.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		sdinotice.hkex.com.hk
```
Comment (J0WI) [inline src/chrome/content/rules/hkex.com.hk.xml]: `http://sdinotice.hkex.com.hk/di/notes/Glossary.htm` (404)

File: src/chrome/content/rules/hkex.com.hk.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+	Time out:
+		hkex.com.hk
+		m.hkex.com.hk
+		search.hkex.com.hk
+-->
```
Comment (J0WI) [inline src/chrome/content/rules/hkex.com.hk.xml]: See also:
`www.iporesults.hkex.com.hk`
`www.esubmission.hkex.com.hk`
`csm.hkex.com.hk`

**Comment by J0WI** [comment]
What about ` www.iporesults.hkex.com.hk`
Can we get rid of the non-capturing group in the redirect rule?

**Comment by Marco129** [comment]
`www.iporesults.hkex.com.hk` is on line 4 marked as `Invalid certificate`

**Comment by J0WI** [comment]
Oh, Thanks!

**Pull request merged**